examples/src/main/scala/com/criteo/cuttle/examples/HelloWorld.scala

```diff
@@ -45,35 +45,49 @@ object HelloWorld {
           val partitionToCompute = (e.context.start) + "-" + (e.context.end)
 
           // The `sh` interpolation is provided by the local platform. It allows us to
-          // declare a bash script to execute.
-          sh"""
-          echo "Hello for ${partitionToCompute}"
-          echo "Check my project page at https://github.com/criteo/cuttle"
-          sleep 1
-        """.exec()
+          // declare a bash command to execute. If you want use many commands you can combine them
+          // in a for comprehension.
+          for {
+            _ <- sh"echo Hello for $partitionToCompute".exec()
+            _ <- sh"echo Check my project page at https://github.com/criteo/cuttle".exec()
+            completed <- sh"sleep 1".exec()
+          } yield completed
```

Comment:
It would be better to log directly instead of forking a process to log.

```
e.streams.info(s"Hello for $partitionToCompute")
e.streams.info(s"Check my project page at https://github.com/criteo/cuttle")
sh"sleep 1".exec()
```

Reply:
I thought about it but for ui's point of view it looks differently like something that came from framework and not the user code. For example it'll be yellow and not white text. Maybe for the sake of example can we keep it that way ?